src/sparsezoo/objects/directory.py

```diff
@@ -284,9 +284,15 @@ def unzip(self, extract_directory: Optional[str] = None, force: bool = False):
             )
 
         name = ".".join(self.name.split(".")[:-2])
-        path = os.path.join(extract_directory, name)
-
-        if not os.path.exists(path) or force:  # do not re-unzip if not forced
+        is_model_gz = name == "model.onnx"
```

Comment:
See other comments, not sure about hard coding this

Reply:
This would work for now, as we are enforcing *decided* constraints on the contents of `model.onnx.tar.gz` but, am open to changing this logic if the constraints are ever relaxed